src/appleseed/renderer/modeling/postprocessingstage/renderstamppostprocessingstage.cpp

```diff
@@ -168,7 +170,8 @@ namespace
                 m_icon.nchannels());
                 
             ImageBuf scaled_logo;
-            ImageBufAlgo::resize(scaled_logo, m_icon, "sharp-gaussian", 1.2f, roi);
+            const float scale_factor = OIIO::lerp(1.0f, 2.8f, (m_scale_factor - MinScaleFactor) / (MaxScaleFactor - MinScaleFactor));
```

Comment:
Does that work for scale factors smaller than 1?

Reply:
Yes it linearly interpolates between filter width 1.0 for the smallest possible user supplied scale factor (0.1) and filter width 2.8 for the largest possible user scale factor (20.0). Here are results for a scale factor of 0.5 set in the GUI: 
![scale_05](https://user-images.githubusercontent.com/26113941/54356636-5fd15280-465c-11e9-9834-025b670e2a51.png)
The const float scale_factor should actually be named filter_width for clarity, can I still change that?

Reply:
Sure!